std_msgs/Header header
float32 score
int32 track_id
float32[4] bbox        # [x, y, w, h] in pixels
geometry_msgs/Point32[4] corners  # TL, TR, BR, BL

